hack/bazel-generate-cache.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+source hack/common.sh
+source hack/bootstrap.sh
+source hack/config.sh
+
+#Creating temp dir
+
+CACHE_DIR=$(mktemp -d)
+
+export BAZEL_CACHE_DIR=$CACHE_DIR
+
+#Bulding with the bazel cache enabled
+bazel build \
+    --config=${ARCHITECTURE} \
+    --disk_cache=$BAZEL_CACHE_DIR \
+    //...
+
+#Cleaning up the temp dir
+rm -rf $CACHE_DIR
```

Comment:
Can you please explain this code? It looks like this would create a temporary directory, use bazel to build with the temp dir as cache target, then the cache is being removed? I'm not sure I understand this logic

Reply:
If I am not mistaken, there might be a configuration issue with bazel, since I believe bazel should do the caching by default on it's own. I would be interested in the findings here.